mode/examples/Topics/File IO/LoadFile2/LoadFile2.pyde

```diff
@@ -36,7 +36,14 @@ def draw():
     for index in xrange(numEntries):
         thisEntry = startingEntry + index
         if thisEntry < len(records):
-            text("%d > %s" % (thisEntry, records[thisEntry].name), 20, 20 + index * 20)
+            text("%d > %s" %
+                (thisEntry,
+                 records[thisEntry].name),
+                20,
+                20 +
+                index *
+                20)
```

Comment:
I tried using autoformat from the PDE and it shows no changes needed. Am i doing something wrong?
![image](https://cloud.githubusercontent.com/assets/12569767/23801008/7c9a8cd0-05d4-11e7-8322-5fa12be217e9.png)